synced-files/.github/labels.yml

```diff
@@ -0,0 +1,123 @@
+# These are Exercism-wide labels which are shared across all repos.
+# We'll use these labels to show "tasks" on the website, which will
+# point users to things they can help contribute with.
+
+# The x:action labels describe what sort of work the contributor
+# will be engaged in when working on the issue
```

Comment:
Could you add a brief explanation that people can add labels via an append? I want to avoid people editing these directly and for now the file isn't code owner-protected (in the long run it probably should be)

Reply:
Done. Could you see if the new explanation makes sense?

Reply:
It does, I left two small suggestions to clarify some things.